scripts/marianasVariables.sh

```diff
@@ -24,5 +26,11 @@ module load gcc/7.3.0
 module load cuda/10.2.89
 module load openmpi/3.1.3
 module load cmake-3.18.4-gcc-7.3.0-fuktvvh
-
-spack env activate exago-v0-99-2-hiop-v0-3-99-2-marianas --with-view
+module load magma-2.5.4-gcc-7.3.0-vgkvbvm
+module load openblas-0.3.12-gcc-7.3.0-dzz6rfy
+module load raja/0.13.0-gcc-7.3.0-pwmbk4o
+module load umpire-4.1.2-gcc-7.3.0-qqotfxd
+module load suite-sparse/5.8.1-gcc-7.3.0-uivxrx7
+module load coinhsl/2015.06.23-gcc-7.3.0-kvdofab
+module load metis-5.1.0-gcc-7.3.0-ymmhgpk
+module load camp-0.1.0-gcc-7.3.0-lfmsuz3
```

Comment:
I would suggest using spack environment here rather than loading spack modulae one by one.

Reply:
CMake links against the environment's hiop library as opposed to the one currently being built - otherwise I would much have rathered the original 😄

Reply:
I believe this is fixable by using correct rpath settings. I'll tinker with this a little more.

Reply:
Rpath isn't so much the issue iiuc - the alternative would be `spack env activate <name> --without-view` and then `spack load -r <dependency>` for every dependency which isn't any more readable than loading modules imo

Reply:
I also don't know how rpath and ld library path are prioritized, so you may be right and the external libhiop might never be found if we better manage our rpaths.

Reply:
I have some new spack environments coming which should make this a lot easier. I think this is good for this PR, but I appreciate the suggestion!